packages/webapp/pages/_app.tsx

```diff
@@ -181,6 +182,7 @@ function InternalApp({ Component, pageProps, router }: AppProps): ReactElement {
         canonical={canonicalFromRouter(router)}
         titleTemplate={unreadCount ? `(${unreadText}) %s` : '%s'}
       />
+      {seo && <NextSeo {...seo} />}
```

Comment:
```suggestion
      {!!seo && <NextSeo {...seo} />}
```

to be safe its always boolean

Reply:
Let me check if there is an eslint to catch these cases automatically 👀